content/topics/java-specific/concurrency/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+content_type: topic
+ready: true
+tags:
+- threads
+- concurrency
+title: Java Multithread, Concurrency and Parallelism
+---
+
+By now, you should have heard about Java being multi-threaded - but what does that mean exactly?
+
+Unlike JavaScript for example - which is a single threaded language; Java has multiple threads to help you run your program faster. 
+
+## What is Multithreading
+
+Multithreading means that you have multiple threads of execution inside the same application. A thread is like a separate CPU executing your application. Thus, a multithreaded application is like an application that has multiple CPUs executing different parts of the code at the same time. 
+
+***A thread goes through a life-cycle which can be seen in the diagram below***
+
+![thread-life-cycle](Thread_Life_Cycle.jpg)
+
+### how to create a thread
+
```

Comment:
Some extra explanation is needed here. Eg:

Java can handle multiple threads, but you need to do a bit of work to set it up. This is because you, as the programmer, need to decide what each thread should be responsible for.